packit_service/worker/helpers/build/copr_build.py

```diff
@@ -481,6 +528,22 @@ def submit_copr_build(self, script: Optional[str] = None) -> Tuple[int, str]:
         """
         owner = self.create_copr_project_if_not_exists()
         try:
+            # TODO: the second check is to be removed in future
+            buildopts = (
+                {
+                    "packit_forge_project": self.forge_project,
+                }
+                if self.is_custom_copr_project_defined
```

Comment:
I can see some [incorrect messages in staging](https://github.com/packit/hello-world/pull/654#issuecomment-1253181837), maybe the reason is missing `()` here? :thinking:

Reply:
I think you are right! Thanks!